src/AspNet.Security.OAuth.Baidu/BaiduAuthenticationConstants.cs

```diff
@@ -0,0 +1,19 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+namespace AspNet.Security.OAuth.Baidu
+{
+    /// <summary>
+    /// Contains constants specific to the <see cref="BaiduAuthenticationConstants"/>.
```

Comment:
Should this be `BaiduAuthenticationHandler`? Otherwise the comment is just saying it's specific to itself.